lib/rules/sotd/stability.js

```diff
@@ -28,17 +28,21 @@ function findSW(candidates, sr) {
             'A W3C Registry is a specification that, after extensive consensus-building, is endorsed by W3C and its Members.';
     } else {
         const { crType } = sr.config;
-        const INTRO_S =
-            ' A Candidate Recommendation Snapshot has received wide review, is intended to gather implementation experience, and has commitments from Working Group members to royalty-free licensing for implementations.';
-        const INTRO_D =
-            ' A Candidate Recommendation Draft integrates changes from the previous Candidate Recommendation that the Working Group intends to include in a subsequent Candidate Recommendation Snapshot.';
+        const groupIds = sr.getDelivererIDs();
+        const INTRO_S = ` A Candidate Recommendation Snapshot has received wide review, is intended to gather implementation experience, and has commitments from Working Group${
+            groupIds.length > 1 ? 's' : ''
```

Comment:
```suggestion
            groupIds.length > 1 ? "s'" : ''
```

Context: https://github.com/speced/bikeshed-boilerplate/pull/62/files#r1528428357

Reply:
Thanks for checking with comms folks. I'm fine with whatever is grammatically correct, so let's keep the current.